capa/features/freeze/__init__.py

```diff
@@ -194,6 +231,18 @@ class ThreadFeature(HashableModel):
     feature: Feature
 
 
+class CallFeature(HashableModel):
+    """
+    args:
+        call: the call id to which this feature belongs.
+        address: the address at which this feature is found (it's dynamic return address).
```

Comment:
> it's dynamic return address

how can a user reference or use this address?

Reply:
see above: no need to resolve this comment now.

Reply:
> how can a user reference or use this address?

Currently, my interpretation of this is that it's the address associated with that call feature; so for some sandbox, if an argument to an api call is a memory address of a buffer, then the value can be the string in memory, and the address is that in-memory string's location.

Regardless, for CAPE `call` and `address` should be similar, which makes me think that maybe it should be removed for now.